### PR TITLE
docs/ckeditor: add docs for ckeditor

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Comment by phillimorland** [comment]
I don't write much documentation and i'm better with examples then instructions so let me know if it's too vague and i'll try again

File: docs/ckeditor.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# CKEditor
+
+We use the ckeditor in the dashboard in order for users to add formatted text, editable images, accordions and videos. To achieve this we have 4 configured editors for different use cases, default, image-editor, collapsible-image-editor, video-editor.
```
Comment (fuzzylogic2000) [inline docs/ckeditor.md:3]: Do we currently also use the default one without images for ideas?

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! I think it is a perfect start and we can add to it whenever we stumble over more stuff that is confusing or needs explanation.

File: docs/ckeditor.md (hunk 1 above)
Comment (phillimorland) [reply docs/ckeditor.md:3]: yep, we use image upload widget for images in ideas

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!

**Pull request merged**